content/kubeone/master/architecture/operating-system-manager/usage/_index.en.md

```diff
@@ -30,6 +30,56 @@ should be rotated manually, for them to consume configurations generated by OSM.
 
 To perform this rotation please follow the guide at [Rolling Restart MachineDeploments][rolling-restart].
 
+OSM doesn't support using `cloud-init` as provisioning utility for flatcar machines on AWS cloud provider. Flatcar use [coreos-cloudinit](https://github.com/coreos/coreos-cloudinit) instead of the upstream [cloud-init](https://cloud-init.io/). [coreos-cloudinit](https://github.com/coreos/coreos-cloudinit) is deprecated and its usage is not recommended in production environments.
```

Comment:
I'd drop AWS because cloud-init doesn't work on any provider for Flatcar.

```suggestion
OSM doesn't support using `cloud-init` as a provisioning utility for Flatcar machines. Flatcar use [coreos-cloudinit](https://github.com/coreos/coreos-cloudinit) instead of the upstream [cloud-init](https://cloud-init.io/). [coreos-cloudinit](https://github.com/coreos/coreos-cloudinit) is deprecated and its usage is not recommended in production environments.
```